pkg/postgres/schema/process_indicators.go

```diff
@@ -1,6 +1,6 @@
-// Code generated by pg-bindings generator.
+// Code originally generated by pg-bindings generator.
 
-// XXX: Contains manuall changes (extensions for ProcessIndicator schema
```

Comment:
Reference ROX-13790 as TODO (revert to generated code once it is possible to generate multi-column indices).